test/unit/env.js

```diff
@@ -0,0 +1,27 @@
+import jQuery from 'jquery';
+
+global.$ = global.jQuery = jQuery;
+require('bootstrap');
+global.welderApiHost = 'localhost';
+global.welderApiScheme = 'HTTP';
+
+// <mockLocalStorage>
+const localStorageMock = (() => {
```

Comment:
Sorry but I still don't understand why do we need to create the `localStorageMock` object? What will break if we don't create it?